src/sentry/api/event_search.py

```diff
@@ -251,7 +251,8 @@ def visit_raw_search(self, node, children):
             SearchValue(value),
         )
 
-    def visit_numeric_filter(self, node, (search_key, _, operator, search_value)):
+    def visit_numeric_filter(self, node, xxx_todo_changeme):
```

Comment:
What are these changes?

Reply:
I think it was just my linter. I can discard those changes

Reply:
Looks like it happened in a few methods.